CHANGELOG.md

```diff
@@ -68,6 +69,7 @@
 ### ⬆️ Improved
 
 ### ✅ Added
+- Notifications are dismissed after the user go into the channel conversation
```

Comment:
It would be nice to mention this will happen only if you are using our `MessageListViewModel`.
Should we also describe this somewhere in docs? 🤔

Reply:
+1, We should definitely document this somewhere around notifications